tools/vcrpy/vcr/stubs/aiohttp_stubs/__init__.py

```diff
@@ -71,7 +73,7 @@ def build_response(vcr_request, vcr_response, history):
     response._headers = CIMultiDictProxy(CIMultiDict(vcr_response["headers"]))
     response._history = tuple(history)
 
-    response.close()
+    # response.close()
```

Comment:
Can this just be removed? Or are we still testing?

Reply:
In fact, here we applied a patch to vcr 3.0.0. I keep it just to easier track our change. :)

Reply:
Removed.